osu.Game/Screens/Footer/ScreenFooter.cs

```diff
@@ -65,6 +65,8 @@ public ScreenFooter(BackReceptor? receptor = null)
         [BackgroundDependencyLoader]
         private void load()
         {
+            const float footer_button_y_offset = 10;
```

Comment:
I assume this is chosen to be 10 in order to line up with https://github.com/ppy/osu/blob/3ee8eb2809b7570d315e823851214e47e9d64ba7/osu.Game/Screens/Footer/ScreenFooterButton.cs#L28 https://github.com/ppy/osu/blob/3ee8eb2809b7570d315e823851214e47e9d64ba7/osu.Game/Screens/Footer/ScreenFooterButton.cs#L132-L145 - so if we're extracting consts, then why not use that const directly instead?